### ISSUE TITLE
FEAT: Regex in ignores option (component-name-in-template-casing)

### ISSUE THREAD

**Issue by EmmanuelVlad** [vuejs/eslint-plugin-vue#705]
**Feature**
Possibility to add a regex in the `ignores` option

**Example**
```json
{
	"vue/component-name-in-template-casing": ["error", "PascalCase", {
		"ignores": [/v-(.*)/]
	}]
}
```

**Comment by EmmanuelVlad** [comment]
It's kinda fixed for me in #714 but still a good feature to have

**Comment by ota-meshi** [comment]
Changed to be able to set regexp for `ignores` option in PR #714.

**Example**
```json
{
	"vue/component-name-in-template-casing": ["error", "PascalCase", {
		"ignores": ["/v-(.*)/"]
	}]
}
```
